contracts/token/BiconomyTokenPaymaster.sol

```diff
@@ -49,6 +49,18 @@ contract BiconomyTokenPaymaster is
         ORACLE_BASED
     }
 
+    // in case flat fee needs to be enabled
+    // 1. use mode and based on mode treat uint256 fee sent either as priceMarkup or flatFee
+    // 2. (no mode required) add extra value in paymasterandData so uint32 markup and uint224 flatFee both can be parsed
+    // 3. (no mode required) without extra value treat uint256 as packed uint32uint224 and use values accordingly
+    /*enum FeePremiumMode {
+        PERCENTAGE,
+        FLAT
+    }*/
+
+    /// @notice All 'price' variables are multiplied by this value to avoid rounding up
```

Comment:
multiplied or divide? May be better explanation would be

All 'price' variable coming from outside are expected to be multiple of 1e6 coz solidity can't accept floating values, and in actual calculation, final value is divided by PRICE_DENOMINATOR to avoid rounding up.